static/docs/command-reference/import.md

```diff
@@ -1,9 +1,11 @@
 # import
 
-Download or copy file or directory from any <abbr>DVC project</abbr> in a Git
-repository (e.g. hosted on GitHub) into the <abbr>workspace</abbr>, and track
-changes in this [external dependency](/doc/user-guide/external-dependencies).
-Creates [DVC-files](/doc/user-guide/dvc-file-format).
+Import file or directory from any <abbr>DVC project</abbr> in a Git
+repository (e.g. hosted on GitHub). The imported files will be copied
+or downloaded into the <abbr>workspace</abbr> while DVC keeps information
+about the source in a created [DVC-files](/doc/user-guide/dvc-file-format).
+Information about the source is a link to a imported repository and it is
+used during imported file [update](/doc/user-guide/external-dependencies).
```

Comment:
was the intention to link to the `dvc update`?

Reply:
Yes, most likely erroneous copy-paste since the intention of the PR is to remove links to external-dependencies.

Reply:
Fixed.